sw/device/tests/sysrst_ctrl_lib.c

```diff
@@ -0,0 +1,68 @@
+// Copyright lowRISC contributors.
```

Comment:
I think this code should be moved to sw/device/lib/testing/sysrst_ctrl_testutils.[ch].

Reply:
done